### ISSUE TITLE
Android SDK null pointer crash from possible breaking change

### ISSUE THREAD

**Issue by DaveNOTDavid** [Adobe-Marketing-Cloud/acp-sdks#566]
Based on the following crash logs right from registering for the extensions in our app, there may be some breaking changes from your lib updates:
```
java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object[] java.util.Collection.toArray()' on a null object reference
        at java.util.ArrayList.<init>(ArrayList.java:191)
        at com.adobe.marketing.mobile.LegacyMobileConfig.loadMessageImages(LegacyMobileConfig.java:634)
        at com.adobe.marketing.mobile.LegacyMobileConfig.access$200(LegacyMobileConfig.java:47)
        at com.adobe.marketing.mobile.LegacyMobileConfig$2.run(LegacyMobileConfig.java:284)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
```

What we had before for downloading the deps:
```
implementation 'com.adobe.marketing.mobile:mobileservices:1.+'
implementation 'com.adobe.marketing.mobile:target:1.+'
implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
implementation 'com.adobe.marketing.mobile:userprofile:1.+'
```
... which gave us downloaded versions of:
- v1.1.3 for `mobileservices`
- v1.1.6 for `target`
- v1.8.0 for `sdk-core`
- v1.1.0 for `userprofile`

... what struck out is that I don't even see the deployed v1.1.3 version for `mobileservices` in the Maven repo compared to the other extensions' versioning: https://mvnrepository.com/artifact/com.adobe.marketing.mobile/mobileservices

So reverting from v1.1.3 to v1.1.2 worked for us, but I wanted to bring this up to guys for acknowledgement regardless:
```
implementation 'com.adobe.marketing.mobile:mobileservices:1.1.2'
implementation 'com.adobe.marketing.mobile:target:1.1.6'
implementation 'com.adobe.marketing.mobile:sdk-core:1.8.0'
implementation 'com.adobe.marketing.mobile:userprofile:1.1.0'
```

**Comment by rymorale** [comment]
Hi @DaveNOTDavid,

Thanks for the information. We are looking into it and will update here as soon as we have a fix.

**Comment by rymorale** [comment]
Hi @DaveNOTDavid, the fix for the crash has been released in mobile services extension 1.1.4. Closing this issue but please re-open it if needed.

Thanks,
Ryan

**Comment by DaveNOTDavid** [comment]
@rymorale - cool, LGTM now. Thanks 👍
